casper/src/main/scala/coop/rchain/casper/BlockCreator.scala

```diff
@@ -168,7 +168,7 @@ object BlockCreator {
         deploys,
         dag,
         runtimeManager,
-        now,
+        BlockData(now, maxBlockNumber + 1),
```

Comment:
Nitpick: my preference would be to pull this BlockData up a function and pass it through

Reply:
I don't understand. Can you be more explicit?